app/src/main/kotlin/com/simplemobiletools/musicplayer/adapters/AlbumsTracksAdapter.kt

```diff
@@ -130,6 +137,22 @@ class AlbumsTracksAdapter(
         }
     }
 
+    private fun playNext() {
+        getSelectedTracks().firstOrNull()?.let { selectedTrack ->
+            activity.playNextInQueue(selectedTrack) {
+                finishActMode()
+            }
+        }
+    }
+
+    private fun isATrack(): Boolean {
+        getSelectedItems().forEach {
```

Comment:
you can use any() or none() for these

Reply:
Understood. My mistake. Apologies for any unnecessary complications.